src/Modal/_ModalDialog.scss

```diff
@@ -125,6 +125,7 @@
 
   .pgn__modal-title {
     font-size: $h3-font-size;
+    line-height: $h3-font-size * $headings-line-height;
```

Comment:
[inform] Brand packages might override the default font-size/line-height of headings (e.g., `h2` elements). As such, when using `@edx/brand-edx.org`, the `.pgn__modal-title`'s, while the `font-size` is correct per `.pgn__modal-title`, the overridden line-height applied for the `h2` element used for the modal title vs. relying on the changed `h3`'s `line-height`. Explicitly calculating the `line-height` for `.pgn__modal-title` ensures the overridden `h2` element `line-height` does not apply to modal titles.

Reply:
Note: when this change gets ported to v23, the SCSS variables will need to change to the corresponding CSS variables, and likely using `calc(...)`.

[Source](https://github.com/openedx/paragon/blob/31a6a72bc4f731944bb9b79a4a6d1bc8882456bd/src/Modal/_ModalDialog.scss#L144-L148)